themes/agnoster.zsh-theme

```diff
@@ -92,7 +92,7 @@ prompt_git() {
     local LC_ALL="" LC_CTYPE="en_US.UTF-8"
     PL_BRANCH_CHAR=$'\ue0a0'         # 
   }
-  local ref dirty mode repo_path
+  local -a ref dirty mode repo_path
```

Comment:
These are not arrays, so there's no need for it.

Reply:
Reverted my change.